rust/src/shards/gui/layouts/collapsing_header.rs

```diff
@@ -131,13 +137,31 @@ impl Shard for CollapsingHeader {
     Some(&self.requiring)
   }
 
+  fn exposedVariables(&mut self) -> Option<&ExposedTypes> {
```

Comment:
no this is not possible, as the collapsing header can be collapsed and the code path won't be taken.

Reply:
Correct. I will fix it for the *contents* shards but the *header* shards should be able to expose (although that might be a rare case).